### PR TITLE
Fix two TLS things

### PR DESCRIPTION
**Pull request by lolgesten**

### CODE REVIEW

**Comment by algesten** [comment]
@jsha does this look ok to you?

File: tests/https-agent.rs [hunk 1]
```diff
@@ -23,6 +23,8 @@ fn agent_set_header() {
     assert_eq!("value", json.headers.get("Header").unwrap());
 }
 
+// From here https://badssl.com/download/
+// Decrypt key with: openssl rsa -in ./badssl.com-client.pem
```
Comment (jsha) [inline tests/https-agent.rs:27]: Nit: technically this _parses_ the key. It doesn't decrypt it because the key is not encrypted.

**Review by jsha — APPROVED** [review]
Looks good!

File: tests/https-agent.rs (hunk 1 above)
Comment (algesten) [reply tests/https-agent.rs:27]: the key i downloaded from that url had a passphrase that is asked for when running this openssl command. 

is it just obfuscated and not encrypted?

**Pull request merged**